scripts/linkbot/index.js

```diff
@@ -3,14 +3,16 @@ import * as bbMonitor from './monitor-bb.js';
 import * as bbSdMonitor from './monitor-bb-sd.js';
 import * as rpiMonitor from './monitor-rpi.js';
 import * as ubuntuServerMonitor from './monitor-ub-server.js';
-import * as raspbianOsMonitor from './monitor-raspbian-os.js';
+// MEN-5884
+// import * as raspbianOsMonitor from './monitor-raspbian-os.js';
 
 const monitors = {
   bbMonitor,
   bbSdMonitor,
   rpiMonitor,
-  ubuntuServerMonitor,
-  raspbianOsMonitor
+  ubuntuServerMonitor
+  // MEN-5884
+  // raspbianOsMonitor
 };
```

Comment:
We should not need to modify the javascript code - removing the `Auto-update` comment from [here](https://github.com/mendersoftware/mender-convert/blob/5c6783a876e5eb5cc6d8a476af9c0da0cdf3bb2d/.gitlab-ci.yml#L13) should be enough.

Side note to @mzedel, any idea why [this other one](https://github.com/mendersoftware/mender-convert/blob/5c6783a876e5eb5cc6d8a476af9c0da0cdf3bb2d/scripts/test/run-tests.sh#L38) has not being updated by the bot?

Reply:
check http://downloads.raspberrypi.org/raspbian_lite/images/ and see that this is the deprecated raspbian os - there will most likely no longer be any updates for this...

Reply:
> We should not need to modify the javascript code - removing the `Auto-update` comment from [here](https://github.com/mendersoftware/mender-convert/blob/5c6783a876e5eb5cc6d8a476af9c0da0cdf3bb2d/.gitlab-ci.yml#L13) should be enough.

Got it! But I will wait for @mzedel to answer your other question, since it is quite relevant.

Reply:
I think he did @kacf . It is not updated, because there are no new images added to the repo for this image :smile_cat:

Reply:
Right, but then we should manually update it to `raspios` (same as we have in `.gitlab-ci.yml` file) and let the bot update from there on (Note: currently disabling the auto-update due to MEN-5884 but you got my point).

Reply:
Yes, and Yes :)

Reply:
> I think he did @kacf . It is not updated, because there are no new images added to the repo for this image smile_cat

Oh, it takes it from the existing link? Ok, I misunderstood then, I thought the source was hardcoded. Then you're right, everything is answered here. I'll update.